from magicapi.Errors import MagicException


class DoormanAuthException(MagicException):
    pass
